flexx/ui/examples/mondriaan.py

```diff
@@ -33,25 +39,25 @@ def init(self):
             with MyVBox(flex=2):
                 
                 with MyVBox(flex=4, spacing=30):
-                    ui.Widget(flex=1, style='background:#fff;')
-                    ui.Widget(flex=1, style='background:#fff;')
+                    ui.Widget(flex=1, css_class_name='white')
+                    ui.Widget(flex=1, css_class_name='white')
                 
-                with MyVBox(flex=2, style='background:#249;'):
-                    ui.Widget(flex=1, style='background:none;')
-                    ui.Widget(flex=1, style='background:none;')
+                with MyVBox(flex=2, css_class_name='blue'):
+                    ui.Widget(flex=1, css_class_name='edge')
+                    ui.Widget(flex=1, css_class_name='edge')
                 
             with MyVBox(flex=6):
                 
-                with MyVBox(flex=4, spacing=30, style='background:#f23;'):
-                    ui.Widget(flex=1, style='background:none;')
-                    ui.Widget(flex=1, style='background:none;')
+                with MyVBox(flex=4, spacing=30, css_class_name='red'):
+                    ui.Widget(flex=1, css_class_name='edge')
+                    ui.Widget(flex=1, css_class_name='edge')
                 
                 with MyHBox(flex=2):
-                    ui.Widget(flex=6, style='background:#fff;')
+                    ui.Widget(flex=6, css_class_name='white')
                     
                     with MyVBox(flex=1):
-                        ui.Widget(flex=1, style='background:#fff;')
-                        ui.Widget(flex=1, style='background:#ff7;')
+                        ui.Widget(flex=1, css_class_name='white')
+                        ui.Widget(flex=1, css_class_name='yellow')
```

Comment:
This is the first time I've seen [presentational class names](https://seesparkbox.com/foundry/naming_css_stuff_is_really_hard) that are simultaneously also semantic. :)